experimental/material_3_demo/lib/component_screen.dart

```diff
@@ -281,32 +281,61 @@ class FloatingActionButtons extends StatelessWidget {
       label: 'Floating action buttons',
       tooltipMessage:
           'Floating action buttons include: \nFloatingActionButton.small, FloatingActionButton, FloatingActionButton.extended, and FloatingActionButton.large',
-      child: Wrap(
-        alignment: WrapAlignment.spaceEvenly,
-        crossAxisAlignment: WrapCrossAlignment.center,
-        children: [
-          FloatingActionButton.small(
-            onPressed: () {},
-            child: const Icon(Icons.add),
-          ),
-          rowDivider,
-          FloatingActionButton(
-            onPressed: () {},
-            child: const Icon(Icons.add),
-          ),
-          rowDivider,
-          FloatingActionButton.extended(
-            onPressed: () {},
-            icon: const Icon(Icons.add),
-            label: const Text('Create'),
-          ),
-          rowDivider,
-          FloatingActionButton.large(
-            onPressed: () {},
-            child: const Icon(Icons.add),
-          ),
-        ],
-      ),
+      child: LayoutBuilder(builder: (context, constraints) {
+        double screenWidth = MediaQuery.of(context).size.width;
+        if (screenWidth < 375.0) {
+          return Column(
```

Comment:
This works, but seems like a fair amount of duplication. That said, I don't have a cleaner suggestion that wouldn't make it harder to understand. I wonder if instead of just having the extended on a row by itself, would it make sense to just have two rows of two with the small & extended and the medium & large? I am probably overthinking this. Just leave it as it is 😚.

Reply:
Two rows of small & extended and medium & large sounds good to me. Just fixed:)